src/uwtools/logging.py

```diff
@@ -16,6 +17,27 @@
 MSGWIDTH = 69
 
 
+class _Logger:
+    """
+    Support for swappable loggers.
+    """
+
+    def __init__(self) -> None:
+        self.logger = logging.getLogger()  # default to Python root logger.
+
+    def __getattr__(self, attr: str) -> Any:
+        """
+        Delegate attribute access to the currently-used logger.
+
+        :param attr: The attribute to access.
+        :returns: The requested attribute.
+        """
+        return getattr(self.logger, attr)
```

Comment:
This method permits instances of `uwtools.logging._Logger` to respond to arbitrary attribute lookups (e.g. a lookup for an `info` attribute in an expression like `log.info("hello")`) and pass them on to the currently-connected logger.